cmd/zoekt-sourcegraph-indexserver/index.go

```diff
@@ -21,6 +21,10 @@ import (
 	"github.com/google/zoekt/build"
 )
 
+// indexTimeout defines how long the indexserver waits before
+// killing an indexing job.
+const indexTimeout = 1 * time.Hour // an index should never take longer than an hour.
```

Comment:
can we increase this to 1.5h for now? thinking about gigarepo currently being quite slow to index.